www/include/configuration/configCentreonBroker/blockConfig.php

```diff
@@ -45,9 +45,14 @@
  * Cast the block id in int
  */
 try {
-    $id = (string)$_GET['blockId'];
-    $tag = (string)$_GET['tag'];
-    $pos = (int)$_GET['pos'];
+    $id = filter_var((string)$_GET['blockId'], FILTER_SANITIZE_STRING);
```

Comment:
If the blockId parameter does not exist, a notice message (Undefined index) will appear in the PHP log file.

Reply:
those checks are handled l.40

```
if (!isset($_GET['tag']) || !isset($_GET['pos']) || !isset($_GET['blockId'])) {
    exit();
}
```